docs/concepts/models/model_kinds.md

```diff
@@ -261,5 +261,273 @@ FROM db.employees;
 ## SEED
 The `SEED` model kind is used to specify [seed models](./seed_models.md) for using static CSV datasets in your SQLMesh project.
 
+## SCD Type 2
+
+SCD Type 2 is a model kind that supports [slowly changing dimensions](https://en.wikipedia.org/wiki/Slowly_changing_dimension#Type_2:_add_new_row) (SCDs) in your SQLMesh project. SCDs are a common pattern in data warehousing that allow you to track changes to records over time. 
+
+SQLMesh achieves this by adding a `valid_from` and `valid_to` column to your model. The `valid_from` column is the date that the record became valid (inclusive) and the `valid_to` column is the date that the record became invalid (exclusive). The `valid_to` column is set to `NULL` for the latest record.
```

Comment:
what's the format of valid_from and valid_to?

Reply:
They are timestamps so the format is irrelevant. I don't define a precision so I'm curious if we later get feedback about that and wanting to customize but I think it is fine for now.